roles/confluent.test/molecule/rbac-mtls-rhel/verify.yml

```diff
@@ -267,3 +267,20 @@
         service_ca: /var/ssl/private/ca.crt
         service_cert: /var/ssl/private/kafka_connect.crt
         service_key: /var/ssl/private/kafka_connect.key
+
+- name: Validate Cluster Registry
+  hosts: schema_registry
+  gather_facts: false
+  tasks:
```

Comment:
please move the check_cluster_registry.yml tasks here, and remove that file. we can make a generic file later if need be!